server/mongo/models/profileModel.js

```diff
@@ -1,25 +1,34 @@
 const mongoose = require('mongoose');
 
 const profileSchema = new mongoose.Schema({
-    name: {
+    email: {
+        type: String,
+        required: true,
+        unique: true
+    },
+    password: {
         type: String,
         required: true
     },
+    name: {
+        type: String,
+        required: false
+    },
     equipment: {
         type: [String],
-        required: true,
+        required: false,
     },
     interests: {
         type: [String],
-        required: true,
+        required: false,
     },
     location: {
         type: String,
-        required: true,
+        required: false,
     },
     image: {
         type: String,
-        required: true,
+        required: false,
```

Comment:
The sign up workflow creates a profile doc with empty values for the other fields so it should still work. I think the reason it is set to false is because it wasn't accepting empty values.
<img width="377" alt="Screenshot 2023-07-19 at 7 37 22 PM" src="https://github.com/srujanr40/dogfish/assets/54603297/06f25dce-eb59-4231-ab27-ce2740ed4a62">